frontend/widgets/OBSBasic_Clipboard.cpp

```diff
@@ -39,13 +39,13 @@ void OBSBasic::on_actionCopyTransform_triggered()
 void OBSBasic::on_actionPasteTransform_triggered()
 {
 	OBSDataAutoRelease wrapper = obs_scene_save_transform_states(GetCurrentScene(), false);
-	auto func = [](obs_scene_t *, obs_sceneitem_t *item, void *data) {
+	auto func = [](obs_scene_t *, obs_sceneitem_t *item, void * /* data */) {
```

Comment:
```suggestion
	auto func = [](obs_scene_t *, obs_sceneitem_t *item, void *) {
```

In a normal function, I might ask to preserve the argument name. In a lambda, particularly where we already have one parameter without a name, I do not think we need to do this.